packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -262,7 +269,7 @@ class _ReorderableListContentState extends State<_ReorderableListContent> with T
 
   @override
   void didChangeDependencies() {
-    _scrollController = PrimaryScrollController.of(context) ?? ScrollController();
+    _scrollController = widget.scrollController ?? (PrimaryScrollController.of(context) ?? ScrollController());
```

Comment:
The parentheses are probably unnecessary here:
```suggestion
    _scrollController = widget.scrollController ?? PrimaryScrollController.of(context) ?? ScrollController();
```